mewarpx/mewarpx/utils_store/oracle_control.py

```diff
@@ -1,85 +1,112 @@
+"""
```

Comment:
This file has lots of lines that overflow 80 characters some. Probably you just had your text environment set a bit off?